meta-mender-toradex-nxp/README.md

```diff
@@ -5,8 +5,6 @@ Mender integration layer for Toradex family of boards.
 The supported and tested boards are:
 - [Toradex Verdin iMX8M Mini](https://hub.mender.io/t/toradex-verdin-imx8m-mini/2908)
 - [Toradex Verdin iMX8M Plus](https://hub.mender.io/t/toradex-verdin-imx8m-plus/5026)
-- [Toradex Colibri i.MX6ULL](https://hub.mender.io/t/toradex-colibri-i-mx6ull/4102/2)
```

Comment:
Is this remove intentional?

Reply:
This commit was a plain `git revert` of a previous commit  (43d7a6bf235e69b288700c297e15daba9ec7c305), as referenced in the commit message. @drewmoseley was the one that made the original commit.